### DISCUSSION TITLE
End of 2018 Competition Season

### DISCUSSION THREAD

**Discussion by coverbeek** [frc5567/2023-Robot#128] [Team Posts]
Unfortunately, it looks like we’re now officially eliminated from qualifying for States for this year.
http://frc-districtrankings.firstinspires.org/2018/FIM

That’s a bummer, but I want to personally say thank you to all of you for your patience, your hard work, and your contributions to an incredibly successful season. Last year we had much success and made great progress but the code we ran wasn’t as student originated as all of us would have liked. This year we ran ONLY student code, and at East Kentwood, auton worked EVERY SINGLE TIME - 100% success. Every one of you should be proud of that. I certainly am proud of you for that.

Can we do more and better? Yes - and we will. But I want each of you to know that I am extremely proud of the work you did and the quality of the code that was produced. I look forward to building on the great progress made this year and doing even better and more next year!

-Chris

**Comment by 55QA67** [comment]
We still have a chance. Teams often choose not to go to states. If enough teams drip out, we still have a chance
